example-playbooks/run_cleanup/action_plugins/async_alias.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
Why is this file under `run_cleanup`? This looks like something belonging to at least `async_extra`, no?

And in general why all these are under `example-playbooks`? A new module is not an "example". We should think of a better location. 

Something like `scylla-ansible-roles/modules`.

Reply:
> Why is this file under `run_cleanup`? This looks like something belonging to at least `async_extra`, no?

because `run_cleanup` is an independent playbook. there is a special folder structure where ansible looks for modules.

> And in general why all these are under `example-playbooks`? A new module is not an "example". We should think of a better location.

because all our ansible scripts sit in this `example-playbooks` directory. 

> Something like `scylla-ansible-roles/modules`.

yes good idea. we can rename it in a separate patch series.

Reply:
> > Why is this file under `run_cleanup`? This looks like something belonging to at least `async_extra`, no?
> 
> because `run_cleanup` is an independent playbook. there is a special folder structure where ansible looks for modules. probably, I can replace those with soft-links to async-extra if yo uwill.
> 

Yes, that would be better. Let's keep all module related files under the same hierarchy and scripts that use it - under their own separate one.

 Also, I hope there is way to tell the Ansible where to look for extra modules. It would be ridiculous if there isn't.

> > And in general why all these are under `example-playbooks`? A new module is not an "example". We should think of a better location.
> 
> because all our ansible scripts sit in this `example-playbooks` directory.
> 
> > Something like `scylla-ansible-roles/modules`.
> 
> yes good idea. we can rename it in a separate patch series.

I see. Yes. Let's leave it for now here then.

Reply:
removed.
now this can be used: `ANSIBLE_ACTION_PLUGINS=../async_extra/action_plugins ansible-playbook ./run_cleanup.yml -vvvv -l localhost -M ../async_extra/library`.